active_directory/manifest.json

```diff
@@ -27,6 +27,9 @@
     },
     "monitors": {},
     "dashboards": {},
-    "service_checks": "assets/service_checks.json"
+    "service_checks": "assets/service_checks.json",
+    "logs": {
+      "source": ""
```

Comment:
Should we just omit this field so its either the field exists or it doesn't, and we don't have to explicitly check for an empty value? (Keeping just `logs: {}`